dashboard/app/views/plc/learning_modules/show.html.haml

```diff
@@ -3,7 +3,7 @@
 
 %p
   %strong Name:
-  = @learning_module.get_name_with_required_tag
+  = @learning_module.name_with_required_tag
```

Comment:
I didn't build UTs for the learning modules controller at first because there wasn't any real logic there beyond basic CRUD. But as its increased in complexity, UTs seem justified at this point. Something that hits the basic functionality will catch this.